interface/globals.php

```diff
@@ -191,6 +191,10 @@ function undoMagicQuotes($array, $topLevel=true) {
 // Variable set for Eligibility Verification [EDI-271] path
 $GLOBALS['edi_271_file_path'] = $GLOBALS['OE_SITE_DIR'] . "/edi/";
 
+//  Necessary writeable paths for mPDF tool
+define("_MPDF_TEMP_PATH", $GLOBALS['OE_SITE_DIR'].'/documents/mpdf/pdf_tmp/');
+define("_MPDF_TTFONTDATAPATH", $GLOBALS['OE_SITE_DIR'].'/documents/mpdf/ttfontdata/');
```

Comment:
If the directories do not exist, then should create them here. This is important for folks that are using multisite and upgrade (thus would never get those folders into their sites directory. What's neat about this approach is that then won't even need to muck up the official repo with these empty folders.